src/reference/supported-platforms.md

```diff
@@ -40,10 +40,10 @@ following table:
 |Platform version|Supported|Best effort|Unsupported|
 |----------------|---------|-----------|-----------|
 | Android SDK    |21-34|19-20|18-|
-| iOS            |16|11-15|10-, arm7v 32-bit|
+| iOS            |16|11-15, 17|10-, arm7v 32-bit|
```

Comment:
iOS 17 should be supported, not best effort.

Reply:
Same here, came from the iOS team, I think because the release had minimal CI testing on iOS 17.

Reply:
iOS 17 is apparently already installed on a 1/3rd of devices. https://telemetrydeck.com/blog/ios-market-share-10-23/

Reply:
Yeah I definitely see why you see it as important to have it there. Developers would definitely like support to be signaled.

We just currently define **Supported** as "Google-tested platforms that are automatically tested on every commit by continuous integration testing."

Since 3.16 was cut before iOS 17 release, I'm just assuming it had limited automatic testing on the platform version. I'm not too familiar with the Flutter testing setup nor these support levels. Maybe we can pull some others in who are.

Do you think "Best effort" is too off putting? Maybe we can consider different names for the levels of support.